### PR TITLE
feat: Scene-by-Fragment for Vixen Network

### PR DESCRIPTION
**Pull request by TgSeed**
feat: Scene-by-Fragment for Vixen Network
feat: Vixen Network Shows the exact match in search if possible *

* Requires scene date in filename

### CODE REVIEW

File: scrapers/vixenNetwork.py [hunk 1]
```diff
@@ -244,24 +283,47 @@ def sortByLength(sites):
...
+else:
+    stdin = sys.stdin.read()
+
+frag = json.loads(stdin)
 search_query = frag.get("name")
 url = frag.get("url")
-
-#sceneByURL
+filename = frag.get("path")
+# path is available in my fork and might be available in future stash versions.
```
[the remark this answers is not in the document]
Comment (TgSeed) [reply scrapers/vixenNetwork.py]: Thanks, didn't knew of that file and function.

**Comment by TgSeed** [comment]
The scraper has got a lot better by the [b135df2](https://github.com/stashapp/CommunityScrapers/pull/1035/commits/b135df22fdc51f31e09b223ed337db92a43a12c7) commit.

**Comment by TgSeed** [comment]
The scraper has got a lot better by the [4b6f801](https://github.com/stashapp/CommunityScrapers/pull/1035/commits/4b6f801a63575fc5eabefe250cabe0706dacf0a1) commit.

**Comment by Maista6969** [comment]
Closing this as it adds the same functionality and thus conflicts with #1582

**Pull request closed without merging**